content/docs/guides/continuous-delivery/github-actions.md

```diff
@@ -239,47 +273,109 @@ on:
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-region: ${{ env.AWS_REGION }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      - name: Install Pulumi CLI
+        uses: pulumi/action-install-pulumi-cli@v1.0.1
+      - run: npm install
+      - uses: pulumi/actions@v2
+        with:
+          command: preview
+          comment-on-pr: true
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          work-dir: infra
+          stack-name: dev
+        env:
+          PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
+```
 
 Note that you'll need to create these stacks [in the usual
 way]({{< relref "/docs/intro/concepts/stack" >}}) using the Pulumi Console or CLI.
 After setting this up, everything will be on autopilot.
 
-## Demos and Examples
+## Migrating from GitHub Action v1?
+
+Here are some pointers when migrating from v1 to v2 of our GitHub Action.
+
+* The following inputs have changed from environment variables to action inputs:
+    * `PULUMI_ROOT` is now `work-dir`
+    * `PULUMI_BACKEND_URL` is now `cloud-url`
+    * `COMMENT_ON_PR` is now `comment-on-pr`
+    * `GITHUB_TOKEN` is now `github-token`
 
-To see some examples of this in action, see the following links:
+* `IS_PR_WORKFLOW` is no longer a viable input. The action is able to understand if the workflow is a pull_request due to action type
 
-* [Our introductory blog post]({{< relref "continuous-delivery-to-any-cloud-using-github-actions-and-pulumi" >}})
-* [Dockerized Ruby on Rails, in Kubernetes, with hosted Cloud SQL](https://github.com/pulumi/actions-example-gke-rails)
-* [Short 90 second video from GitHub Universe Keynote](https://www.youtube.com/watch?time_continue=56&v=59SxB2uY9E0)
-* [Short 90 second video on GitOps and Pull Request workflows](https://www.youtube.com/watch?v=MKbDVDBuKUA)
-* [Longer 7 minute video exploring the ins and outs of Pulumi GitHub Actions in practice](https://www.youtube.com/watch?time_continue=1&v=1Et2TkuxqJg)
+* The action now runs natively, so the action workflow needs to have the correct environment configured. There
+  are sample workflows available in out [Actions repository](https://github.com/pulumi/actions/tree/master/.github/workflows).
+  For examples, if you are running a NodeJS (for example) app then you need to ensure that your action has NodeJS available to it:
+
+```yaml
+- uses: actions/setup-node@v1
+  with:
+    node-version: 14.x
+```
+
+* The action will no longer run `npm ci | npm install | pip3 install | pipenv install`. Please ensure that you are installing
+  your dependencies before Pulumi commands are executed, e.g.:
+
+```
+- run: pip install -r requirements
+  working-directory: infra
+```
```

Comment:
Since this code block is associated with the preceding bullet point, it should be nested/indented under it.

```suggestion
    ```
- run: pip install -r requirements
  working-directory: infra
    ```
```